src/core/fonts.js

```diff
@@ -6883,6 +6963,14 @@ var CFFFDSelect = (function CFFFDSelectClosure() {
     this.fdSelect = fdSelect;
     this.raw = raw;
   }
+  CFFFDSelect.prototype = {
+    getFDIndex: function CFFFDSelect_get(glyphIndex) {
+      if (glyphIndex < 0 || glyphIndex >= this.fdSelect.length) {
+        return false;
```

Comment:
Based on the name of the function, you might expect it to always return an integer, so would it make sense to instead return `-1` in this case?